azure-pipelines.yml

```diff
@@ -12,7 +12,7 @@ variables:
   buildConfiguration: 'Release'
   buildPlatform: 'x64'
   major: 0
-  minor: 5
+  minor: 6
```

Comment:
Do we need change version tags in arm template?

Reply:
I plan to create the 0.6.0 image on our public ARC, then update the version and publish the github release.